charts/trino/values.yaml

```diff
@@ -727,6 +781,71 @@ worker:
   #    path: /secrets/sample.json
   # ```
 
+  jmx:
+    # -- Set to true to enable the RMI server to expose the workers' [JMX metrics](https://trino.io/docs/current/admin/jmx.html).
+    enabled: false
+    registryPort: 9080
+    serverPort: 9081
+    exporter:
+      # worker.jmx.exporter.enabled -- Set to true to export JMX Metrics via HTTP for [Prometheus](https://github.com/prometheus/jmx_exporter) consumption
+      enabled: false
+      image: bitnami/jmx-exporter:latest
+      pullPolicy: Always
+      port: 5556
+      configProperties: ""
+      # worker.jmx.exporter.configProperties -- The string value is templated using `tpl`. JMX Config Properties is mounted to /etc/jmx-exporter/jmx-exporter-config.yaml
+      # @raw
+      # Example:
+      # ```yaml
+      #  configProperties: |-
+      #     hostPort: localhost:{{- .Values.worker.jmx.registryPort }}
+      #     startDelaySeconds: 0
+      #     ssl: false
+      #     lowercaseOutputName: false
+      #     lowercaseOutputLabelNames: false
+      #     includeObjectNames: ["java.lang:type=Threading"]
+      #     autoExcludeObjectNameAttributes: true
+      #     excludeObjectNameAttributes:
+      #       "java.lang:type=OperatingSystem":
+      #         - "ObjectName"
+      #       "java.lang:type=Runtime":
+      #         - "ClassPath"
+      #         - "SystemProperties"
+      #     rules:
+      #       - pattern: 'java\.lang<type=Threading><(.*)>ThreadCount: (.*)'
+      #         name: java_lang_Threading_ThreadCount
+      #         value: '$2'
+      #         help: 'ThreadCount (java.lang<type=Threading><>ThreadCount)'
+      #         type: UNTYPED
+      securityContext: {}
+      resources: {}
+      # worker.jmx.exporter.resources -- It is recommended not to specify default resources
+      # and to leave this as a conscious choice for the user. This also increases
+      # chances charts run on environments with little resources, such as Minikube.
+      # If you do want to specify resources, use the following example, and adjust
+      # it as necessary.
+      # @raw
+      # Example:
+      # ```yaml
+      #  limits:
+      #    cpu: 100m
+      #    memory: 128Mi
+      #  requests:
+      #    cpu: 100m
+      #    memory: 128Mi
+      # ```
+
+  podMonitor:
```

Comment:
If there would be a headless service created for workers, can we keep using only service monitors? That might simplify the chart, and a headless service could be useful for other stuff, like in SANs of custom TLS certs used for internal cluster communication.

Reply:
Makes sense, we can also use the worker service for tests when we need to communicate with workers directly.